LICENSE

```diff
@@ -0,0 +1,233 @@
+Dual License:
+=============
+
+- Eclipse Distribution License - v 1.0
+https://www.eclipse.org/org/documents/edl-v10.php
+
+or
+- Eclipse Public License - v 1.0
+https://www.eclipse.org/org/documents/epl-v10.php
+
```

Comment:
The [Eclipse Project Handbook](https://eclipse.org/projects/handbook/#legaldoc) proposes some explicit wording to use for a LICENSE file for a dual license project:
````
This program and the accompanying materials are made available under the
terms of the Eclipse Public License 2.0 which is available at
http://www.eclipse.org/legal/epl-2.0, or the Apache Software License
2.0 which is available at https://www.apache.org/licenses/LICENSE-2.0.

... Text of the Eclipse Public License 2.0 ...

... Text of the Apache Software License 2.0 ...
````

maybe we can adapt that wording?

Reply:
And replace the "apache" with EDL?

Reply:
of course, that's the alternative license we offer, isn't it?